operator/builtin/output/newrelic/newrelic_test.go

```diff
@@ -64,21 +64,21 @@ func TestNewRelicOutput(t *testing.T) {
 			nil,
 			[]*entry.Entry{{
 				Timestamp: time.Date(2016, 10, 10, 8, 58, 52, 0, time.UTC),
-				Record:    "test1",
+				Body:      "test1",
 			}, {
 				Timestamp: time.Date(2016, 10, 10, 8, 58, 52, 0, time.UTC),
-				Record:    "test2",
+				Body:      "test2",
 			}},
 			`[{"common":{"attributes":{"plugin":{"type":"stanza","version":"unknown"}}},"logs":[{"timestamp":1476089932000,"attributes":{"labels":null,"record":"test1","resource":null,"severity":"default"},"message":"test1"},{"timestamp":1476089932000,"attributes":{"labels":null,"record":"test2","resource":null,"severity":"default"},"message":"test2"}]}]` + "\n",
```

Comment:
```suggestion
			`[{"common":{"attributes":{"plugin":{"type":"stanza","version":"unknown"}}},"logs":[{"timestamp":1476089932000,"attributes":{"labels":null,"record":"test1","resource":null,"severity":"default"},"message":"test1"},{"timestamp":1476089932000,"attributes":{"labels":null,"record":"test2","resource":null,"severity":"default"},"message":"test2"}]}]` + "\n",
```
my only other thing I'm seeing is that this should be body but the tests pass sooooo not sure.

Reply:
@djaglowski Am I not seeing something here? shouldn't the labels and record field be changed?

Reply:
I'm investigating this. I just ran the tests with your suggested changes and it passed. Seems like it should now have worked both ways.

Reply:
Ok tests are failing. Looks like in `newrelic/payload.go::LogMessageFromEntry` it was being hard coded into a map.

Reply:
@djaglowski Does changing the fields in NewRelic from `record` -> `body` and `labels` -> `attributes` break backwards compatibility with any customers who may have built things off those fields?

Since this is for v2 do we care as it's a part of an overall breaking change?

Reply:
I'm not sure if anyone is using the newrelic operator, but it seems unlikely. As you pointed out, the major version gives us some liberty to break things if necessary though. I support making the change now.